### PR TITLE
chore: Add more remap examples for remap conditions

### PR DESCRIPTION
**Pull request by binarylogic**
Signed-off-by: binarylogic <bjohnson@binarylogic.com>

### CODE REVIEW

File: docs/reference/components/transforms/reduce.cue [hunk 1]
```diff
@@ -135,7 +135,90 @@ components: transforms: reduce: {
...
+						message: "Hello world, I am a new log",
+						host: "host-1.hostname.com"
+						pid: 1234
+						tid: 5678
+					}
+				}
+			]
+		},
+		{
+			title: "Reduce Rails logs into a signle transaction"
```
Comment (jamtur01) [inline docs/reference/components/transforms/reduce.cue:221]: single

**Review by jamtur01 — CHANGES_REQUESTED** [review]
One spelling issue.

File: docs/reference/components/transforms/reduce.cue [hunk 2]
```diff
@@ -135,7 +135,90 @@ components: transforms: reduce: {
...
+						tid: 5678
+					}
+				},
+			]
+			configuration: {
+				group_by: ["host", "pid", "tid"]
+				marge_strategies: message: "concat_newline"
+				starts_when: {
+					type: "remap"
+					source: 'match(.message, /^[^\s]/)'
```
Comment (bruceg) [inline docs/reference/components/transforms/reduce.cue:191]: Does this `\s` need an extra `\` in the Cue source?

Comment (binarylogic) [reply docs/reference/components/transforms/reduce.cue:191]: I don't think so. It was necessary when using non-literal strings in TOML, hence the `'` characters.

**Pull request merged**